src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -24,54 +23,52 @@
 
     public static final String COMMAND_WORD = "add";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds an entry (revenue or expense) to " +
+                "Common Cents\n"
             + "Parameters: "
-            + PREFIX_NAME + "NAME "
-            + PREFIX_PHONE + "PHONE "
-            + PREFIX_EMAIL + "EMAIL "
+            + PREFIX_CATEGORY + "CATEGORY "
+            + PREFIX_DESCRIPTION + "PHONE "
+            + PREFIX_AMOUNT + "EMAIL "
```

Comment:
Can change thr Strings here, like "PHONE " to "DESCRIPTION " and same for amount